<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gy801</class>
 <widget class="QMainWindow" name="gy801">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gy801</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>800</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>800</height>
    </size>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>720</y>
      <width>461</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="start_layout">
     <item>
      <widget class="QPushButton" name="button_packet">
       <property name="text">
        <string>PACKET STRUCT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_COMHint">
       <property name="text">
        <string>COM-port:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_edit_COM">
       <property name="text">
        <string>COM4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Baud rate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_edit_baud">
       <property name="text">
        <string>9600</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_start">
       <property name="text">
        <string>START</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>721</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="render_layout"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>720</y>
      <width>341</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="console_layout"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
